python/rpdk/java/codegen.py

```diff
@@ -128,69 +223,63 @@ def init(self, project):
         )
         project.safewrite(path, contents)
 
-        LOG.debug("Writing handlers and tests")
-        self.init_handlers(project, src, tst)
-
-        LOG.debug("Writing callback context")
-        template = self.env.get_template("CallbackContext.java")
-        path = src / "CallbackContext.java"
-        contents = template.render(package_name=self.package_name)
-        project.safewrite(path, contents)
-
-        path = src / "Configuration.java"
-        LOG.debug("Writing configuration: %s", path)
-        template = self.env.get_template("StubConfiguration.java")
-        contents = template.render(
-            package_name=self.package_name,
-            schema_file_name=project.schema_filename,
-            pojo_name="ResourceModel",
-        )
-        project.safewrite(path, contents)
-
-        # generated docs
+        # generate docs
         path = project.root / "README.md"
         LOG.debug("Writing README: %s", path)
-        template = self.env.get_template("README.md")
+        template = self.env.get_template("init/shared/README.md")
         contents = template.render(
             type_name=project.type_name,
             schema_path=project.schema_path,
             executable=EXECUTABLE,
         )
         project.safewrite(path, contents)
 
-        LOG.debug("Init complete")
+    @logdebug
+    def init_default(self, project, src, tst):
+        """Writing handlers and tests"""
+        self.init_handlers(project, src, tst)
+
+    @logdebug
+    def init_guided_aws(self, project, src, tst):
+        """Writing handlers and tests"""
+        self.init_handlers(project, src, tst)
```

Comment:
This could really be part of `init_shared` or bubbled up to avoid the duplication. Any reason not to? Is this what was causing the pylint error?